app/src/pages/lloydGeorgeUploadPage/LloydGeorgeUploadPage.test.tsx

```diff
@@ -60,8 +76,11 @@ jest.mock(
     '../../components/blocks/lloydGeorgeUploadFailedStage/LloydGeorgeUploadFailedStage',
     () => () => <h1>Mock file failed stage</h1>,
 );
+jest.mock('react-router', () => ({
+    useNavigate: () => mockNavigate,
+}));
 
-describe('UploadDocumentsPage', () => {
+describe('LloydGeorgeUploadDocumentsPage', () => {
```

Comment:
```suggestion
describe('LloydGeorgeUploadPage', () => {
```
maybe better to put the component name here, same as other tests